pyrasterframes/python/tests/PyRasterFramesTests.py

```diff
@@ -111,8 +196,13 @@ def test_tile_functions(self):
             .withColumn('expm1', rf_expm1(self.tileCol)) \
             .withColumn('round', rf_round(self.tileCol)) \
             .withColumn('abs', rf_abs(self.tileCol))
+        # TODO: add test for rf_extent and rf_geometry once rastersource connector is integrated and we have
+        #  a source of ProjectedRasterTiles.
+        df.first()
 
-        df.show()
+    def test_agg_mean(self):
+        mean = self.rf.agg(rf_agg_mean(self.tileCol)).first()['rf_agg_mean(tile)']
+        self.assertTrue(self.rounded_compare(mean, 10160))
 
     def test_prt_functions(self):
```

Comment:
Suggest this actually go into the RasterSource class

Reply:
👍